### PR TITLE
Implement Command enchantment

### PR DESCRIPTION
**Pull request by dvitski**
https://user-images.githubusercontent.com/36608902/191047886-56926958-3694-4fa1-ae33-ee434a2b7129.mp4

https://user-images.githubusercontent.com/36608902/191048194-5d37b1ba-4ed7-4dea-ba45-d05df8449fb0.mp4

### CODE REVIEW

File: common/src/main/java/com/github/teamfusion/spyglassplus/client/network/SpyglassPlusClientNetworking.java [hunk 1]
```diff
@@ -28,12 +33,35 @@ public interface SpyglassPlusClientNetworking extends SpyglassPlusNetworking {
...
         NetworkManager.registerReceiver(Side.S2C, INDICATE_UPDATE_PACKET_ID, SpyglassPlusClientNetworking::onIndicateUpdate);
+        NetworkManager.registerReceiver(Side.S2C, COMMAND_UPDATE_PACKET_ID, SpyglassPlusClientNetworking::onCommandUpdate);
+        NetworkManager.registerReceiver(Side.S2C, COMMAND_TARGETED_PACKET_ID, SpyglassPlusClientNetworking::onCommandTargeted);
+    }
+
+    /**
+     * Receives an entity's effects for {@link SpyglassPlusEnchantments#DISCOVERY}.
+     */
+    static void onDiscoveryEffectsUpdate(PacketByteBuf buf, PacketContext context) {
+        int id = buf.readInt();
```
Comment (dvitski) [inline common/src/main/java/com/github/teamfusion/spyglassplus/client/network/SpyglassPlusClientNetworking.java:44]: Messy diff due to rearranging the methods, don't worry about this

File: common/src/main/java/com/github/teamfusion/spyglassplus/mixin/client/KeyBindingMixin.java [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+        return value;
+    }
+
+    /* Custom Logic */
+
+    @Inject(method = "onKeyPressed", at = @At("HEAD"))
+    private static void onOnKeyPressed(InputUtil.Key key, CallbackInfo ci) {
+        KeyBinding keyBinding = SpyglassPlusKeyBindings.TRIGGER_COMMAND_ENCHANTMENT;
+        KeyBindingAccessor access = (KeyBindingAccessor) keyBinding;
+        if (access.getBoundKey() == key) access.setTimesPressed(access.getTimesPressed() + 1);
```
Comment (Hellion-Games) [inline common/src/main/java/com/github/teamfusion/spyglassplus/mixin/client/KeyBindingMixin.java]: One-lined if

File: common/src/main/java/com/github/teamfusion/spyglassplus/mixin/client/KeyBindingMixin.java [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+    }
+
+    @Inject(method = "setKeyPressed", at = @At("HEAD"))
+    private static void onSetKeyPressed(InputUtil.Key key, boolean pressed, CallbackInfo ci) {
+        KeyBinding keyBinding = SpyglassPlusKeyBindings.TRIGGER_COMMAND_ENCHANTMENT;
+        KeyBindingAccessor access = (KeyBindingAccessor) keyBinding;
+        if (access.getBoundKey() == key) keyBinding.setPressed(pressed);
```
Comment (Hellion-Games) [inline common/src/main/java/com/github/teamfusion/spyglassplus/mixin/client/KeyBindingMixin.java]: Another one-lined if. Probably just slipped by you b/c I noticed an expanded version above.

**Review by Dariensg — CHANGES_REQUESTED** [review]
Just a couple one-lined if statements I caught. Once you fix that, you're all set to merge

**Pull request merged**